care/utils/evaluators/evaluation_metric/validators.py

```
@@ -0,0 +1,53 @@
+from pydantic import UUID4
+
+
+def validate_patient_age_equality(value):
+    if not isinstance(value, (int, float)):
+        raise ValueError("patient_age equality operation requires an numeric value")
```

Ruff (0.13.1)

Prefer `TypeError` exception for invalid type

(TRY004)